c/meson.build

```diff
@@ -192,6 +218,14 @@ configure_file(input: 'capi-ml-inference-pipeline.pc.in', output: 'capi-ml-infer
   configuration: ml_inf_pipe_conf
 )
 
+ml_inf_service_conf = configuration_data()
+ml_inf_service_conf.merge_from(api_conf)
+ml_inf_service_conf.set('ML_INFERENCE_SERVICE_REQUIRE', 'capi-ml-inference-pipeline')
```

Comment:
Q: does it require only pipeline? (or both single and pipeline)

Reply:
The current implementation does not require both pipeline and single dependencies since newly added APIs are only related to DB operation. However, they will be required for the next version so I will add both of them.